api/FontFace.json

```
@@ -240,24 +240,40 @@
           }
         }
       },
-      "style": {
+      "load": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FontFace/style",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FontFace/load",
           "support": {
-            "chrome": {
-              "version_added": null
-            },
-            "chrome_android": {
-              "version_added": null
-            },
+            "chrome": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "35",
+                "version_removed": "45",
+                "partial_implementation": true,
+                "notes": "Starting in Chrome 45, the returned promise resolves with a <code>FontFace</code> object as required by the specification instead of void."
```

The notes here are a little odd (it's a note about the version 45 implementation, not the version 35 implementation). I think there are two ways to improve this. Here's my preference:

```suggestion
                "notes": "The returned promise resolves void instead of a <code>FontFace</code> object as required by the specification."
```

Alternatively, the note can be moved to the 45 entry above.